lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -1071,25 +1071,20 @@ def get_issued_certificates(
         Returns:
             dict: Certificates per application name.
         """
-        certificates: Dict[str, Dict[str, str]] = defaultdict(dict)
+        certificates: Dict[str, str] = defaultdict(str)
         relations = (
-            [
-                relation
-                for relation in self.model.relations[self.relationship_name]
-                if relation.id == relation_id
-            ]
-            if relation_id is not None
+            [self.model.relations[self.relationship_name][relation_id]]
+            if relation_id
             else self.model.relations.get(self.relationship_name, [])
         )
         for relation in relations:
             provider_relation_data = _load_relation_data(relation.data[self.charm.app])
             provider_certificates = provider_relation_data.get("certificates", [])
             for certificate in provider_certificates:
-                if not certificate.get("revoked", False):
-                    certificates[relation.app.name].update(  # type: ignore[union-attr]
-                        {certificate["certificate_signing_request"]: certificate["certificate"]}
-                    )
+                certificates[relation.app.name] = f"CSR: \n{certificate['certificate_signing_request']}\nCERT: \n{certificate['certificate']}"
```

Comment:
Putting a note here to link to related discussion: https://github.com/canonical/self-signed-certificates-operator/pull/25#discussion_r1302733037

Reply:
(I don't know if that's the same comment as @DanielArndt above)

I'm unconvinced that this changes improves the lib in any way. Here we're returning a string of a format that is unknown to users of the lib. If there is data manipulation to be done, let's make it at the implementation/charm level, not in the lib itself.

Reply:
Also, why get rid of the `if not revoked` condition?